deltacat/compute/compactor/model/compaction_session_audit_info.py

```diff
@@ -902,6 +927,7 @@ def save_round_completion_stats(
         self.set_output_file_count(pyarrow_write_result.files)
         self.set_output_size_bytes(pyarrow_write_result.file_bytes)
         self.set_output_size_pyarrow_bytes(pyarrow_write_result.pyarrow_bytes)
+        self.set_output_record_count(pyarrow_write_result.records)
```

Comment:
Probably need to aggregate `untouched_record_count` with total output_record_count if you want to get a total of record count in the Ray compacted table?

Reply:
We can do that calculation in the driver under the driver audit entry with the logic resembling:

```py
net_output_records = 0
for each partition_compaction_audit:
     net_output_records += (partition_compaction_audit.output_record_count + partition_compaction_audit.untouched_record_count)
OUTPUT_RECORDS = net_output_records
```